Runtime/Scripts/BaseMeshSync.cs

```diff
@@ -788,6 +788,10 @@ internal void AfterUpdateScene()
             }
 #endif
 
+#if VERBOSE_LOGS
```

Comment:
If we have plenty of these, I'd suggest using conditional attribute similar to what mloccy did in https://github.com/unity3d-jp/MeshSync/pull/807

Reply:
Good idea, I created a class for it now.